docs/SEEEDSTUDIO_WIO_E5.md

```diff
@@ -44,14 +44,11 @@ Connections (name in respect to board print-ons):
 - led red: on-board
 - bind button: BOOT button, hold for 4 seconds to initiate bind mode
 
-If you want to communicate with the radio via the JR bay (pin 5 in the JR bay), then you in addition need to do:
-
-- solder a Schottky diode (e.g. BAT42) between pads RX1 and TX1 (RX1 - |<| - TX1, |<| represents the diode)
-- use firmware with DEVICE_HAS_JRPIN5 enabled (this is the default)
-
 Example Wiring using JR bay with CRSF:
 
-<img src="images/E5_Mini_Tx_Wiring.png" width="800px">
+<img src="images/E5_Mini_Tx_Wiring.png" width="600px">
+
+*The Schottky diode between the TX1 and RX1 pins is not normally needed and this configuration uses the default firmware (labeled tx-Wio-E5-Mini-wle5jc-vX.X.XX.hex).  If you are having communication issues between the radio and the module then you can install the Schottky diode and use the sdiode firmware (labeled tx-Wio-E5-Mini-wle5jc-sdiode-vX.X.XX.hex).  
```

Comment:
maybe "... issues between the radio and the Tx module ..."
mabye one should put sdiode into quotation marks ?
what happens if we would add e.g. a "-in" version which offers a sbus/crsf input instead of the jrpin5 functionality? Generally, what happens if there are more firmware variants? E.g. in case of the E77-MBL we have -400 and -900...

Reply:
Updated language to include Tx, handle multiple firmware revisions.

Reply:
👍